tests/toolchain/install.pm

```diff
@@ -24,23 +24,35 @@ sub run {
     # disable packagekitd
     script_run 'systemctl mask packagekit.service';
     script_run 'systemctl stop packagekit.service';
-    # toolchain channels
-    if (!check_var('ADDONS', 'tcm')) {
-        my $arch = get_var('ARCH');
-        assert_script_run "zypper ar -f http://download.suse.de/ibs/SUSE/Products/SLE-Module-Toolchain/12/$arch/product/ SLE-Module-Toolchain12-Pool";
-        assert_script_run "zypper ar -f http://download.suse.de/ibs/SUSE/Updates/SLE-Module-Toolchain/12/$arch/update/ SLE-Module-Toolchain12-Updates";
+
+    if (is_sle && !sle_version_at_least('15')) {
+        # toolchain channels
+        if (!check_var('ADDONS', 'tcm')) {
+            my $arch = get_var('ARCH');
+            assert_script_run "zypper ar -f http://download.suse.de/ibs/SUSE/Products/SLE-Module-Toolchain/12/$arch/product/ SLE-Module-Toolchain12-Pool";
+            assert_script_run "zypper ar -f http://download.suse.de/ibs/SUSE/Updates/SLE-Module-Toolchain/12/$arch/update/ SLE-Module-Toolchain12-Updates";
+        }
+        zypper_call('in -t pattern gcc5');
+        zypper_call('up');
+
+        # reboot when runing processes use deleted files after packages update
+        type_string "zypper ps|grep 'PPID' || echo OK | tee /dev/$serialdev\n";
+        if (!wait_serial("OK", 100)) {
+            type_string "shutdown -r now\n";
+            $self->wait_boot;
+            select_console('root-console');
+        }
+        script_run 'export CC=/usr/bin/gcc-5';
+        script_run 'export CXX=/usr/bin/g++-5';
     }
-    zypper_call('in -t pattern gcc5');
-    zypper_call('up');
-    # reboot when runing processes use deleted files after packages update
-    type_string "zypper ps|grep 'PPID' || echo OK | tee /dev/$serialdev\n";
-    if (!wait_serial("OK", 100)) {
-        type_string "shutdown -r now\n";
-        $self->wait_boot;
-        select_console('root-console');
+    elsif (is_sle) {
+        # there is no pattern for sle-15 bsc#1070151
+        # gcc + gcc-fortran from Base module & patch for Application module
+        zypper_call 'in gcc gcc-fortran patch';
```

Comment:
Doesn't the bug state that "patterns-devel-base" should be used?

Reply:
Sure, I can add it, even if it is not strictly needed, for future development I guess, but anyway the gcc-fortran need to be included, is not part of the pattern.
Verification run for SLE15: http://dhcp227/tests/604